### DISCUSSION TITLE
Mac installation

### DISCUSSION THREAD

**Discussion by moodytramp** [itsToggle/plex_debrid#385] [Q&A]
Hey there,
I'm a noob, is there any tutorial for Mac installation ? Can't find it on the main page...

Thanks for the hard work

**Comment by sofiacif** [comment]
Did you manage to find it? I have been trying to find one with no luck

**Reply by moodytramp** [reply]
There is one in the git but I couldn’t manage to do it, I ended up installing it in a raspberry pi

**Reply by sofiacif** [reply]
Is it the docker one? I had never heard of it

**Reply by moodytramp** [reply]
Yeah it's the docker one, it's an app you can install on Mac, but again, it didn't work for me, maybe I messed up at some point. If you manage to do it, please let me know

**Reply by sofiacif** [reply]
thank you for your quick answer! I'm getting this error:
Error response from daemon: create realdebrid: error looking up volume plugin rclone: plugin rclone found but disabled

Maybe i'm also doing something wrong
